### DISCUSSION TITLE
Fake-08 Anbernic RG35XX-H

### DISCUSSION THREAD

**Discussion by Goldilocks725** [jtothebell/fake-08#233] [Q&A]
Hello,

I'm using Anbernic RG35XX-H with stock OS firmware version RG35xxH-20240130.
![20240408_225309](https://github.com/jtothebell/fake-08/assets/166421742/061f6bf0-87ca-468c-b409-10bdd0deae50)

I want to ask why the Marble Merger doesnt work on Fake-08? I have tried to change the ext either png or p8 but still the cart doesnt load properly.

Is there any change on the carts code that I need to update so the fake-08 will recognize the carts?

Sorry, im new here and doesnt really know how this discussion works, any help would be much appreciated. Thank you

**Comment by jtothebell** [comment]
Thank you for the report. I don't have a 35xxh, but it doesn't look like this is an issue specific to it, just something that the emulator doesn't support yet. I'm not sure when I will have time to investigate further, but I will circle back to it at some point. I think I heard that batocera or koriki has proper Pico-8 support, so I would recommend using that on that device.
